README.md

```diff
@@ -138,6 +138,33 @@ var client = new OktaClient(new OktaClientConfiguration
 });
 ```
 
+Key object for assigning to the PrivateKey can be created and initialized inline like in this example for RSA key:
```

Comment:
Can we add more context as to what this is for?  Maybe include a link to more information about why someone would want to do this?

Reply:
I've added the all the info I have to the PR's description